sync-tool/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require": {
+        "php-yaoi/php-yaoi": "1.0.18",
+        "chuyskywalker/rolling-curl": "^3.1"
```

Comment:
Instead of creating a separate composer.json in a subfolder, you could simply use dev requirements in the root composer.json